SauvignonInStardew/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "Name": "Sauvignon in Stardew",
   "Author": "Jesse",
-  "Version": "2.0.0",
+  "Version": "2.0.1-unofficial.1-millerscout",

```

Comment:
(This shouldn't be in the pull request, since it'd add the unofficial version to the official repo.)